src/QirRuntime/experimental/CMakeLists.txt

```diff
@@ -4,8 +4,8 @@
 add_executable(experimental-unittests
   driver.cpp
   IntervalsColoringTests.cpp
-  IonqTranslator.cpp
-  IonqTranslatorTests.cpp
+  CircuitToJasonPrinter.cpp

```

Comment:
nitpick: Jason is usually just for a person's name, while JSON or Json would be used for the format.